### PR TITLE
Unique options

### PR DESCRIPTION
**Pull request by spacecooky**

### CODE REVIEW

File: src/main/java/oc/wh40k/units/or/ORWarboss.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public ORWarboss() {
         waffen = new RuestkammerStarter(ID, randAbstand, cnt, "ORWaffenUndGeschenke", "");
         ((ORWaffenUndGeschenke)waffen.getKammer()).setDefaultFK("Kustom shoota");
         ((ORWaffenUndGeschenke)waffen.getKammer()).setDefaultNK("Big choppa");
-        waffen.initKammer(true,true,false,false,true);
+        waffen.initKammer(true,true,false,false,true,false,true);
```
Comment (chisui) [inline src/main/java/oc/wh40k/units/or/ORWarboss.java]: Could we please use something else then random booleans here? maybe an `EnumSet`?

File: src/main/java/oc/OnlineCodex.java [hunk 2]
```diff
@@ -663,6 +663,15 @@ public void refresh() {
 			
 			double kosten=getKosten();
 			double cp=3+getCP();
+			
+			int  relics = BuildaHQ.getCountFromInformationVectorGlobal("Relic");
+			System.out.println("Relics = " +  relics);
```
[the remark this answers is not in the document]
Comment (spacecooky) [reply src/main/java/oc/OnlineCodex.java]: We could.
All those cases of sysout that are currently still in the code wouldn't be usesful for debugging anyway and should just be removed.

File: src/main/java/oc/wh40k/units/or/ORWarboss.java (hunk 1 above)
Comment (spacecooky) [reply src/main/java/oc/wh40k/units/or/ORWarboss.java]: I'll add it as new task as this should be changed for the RuestkammerStarter in general then.

File: src/main/java/oc/OnlineCodex.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (chisui) [reply src/main/java/oc/OnlineCodex.java]: Yeah, on each startup There just random outputs that don't tell you anything.

File: src/main/java/oc/wh40k/units/or/ORWarboss.java (hunk 1 above)
Comment (chisui) [reply src/main/java/oc/wh40k/units/or/ORWarboss.java]: Ok, Then it's ok for now I guess.

**Pull request merged**